backend/database/migrations/2021_08_02_005225_create_students_table.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->string('email');
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
-            $table->string('thumbnail')->nullable();
+            $table->string('thumbnail')->default('/storage/app/thumbnails/defaultProfile.jpg');
```

Comment:
instead of sa previous migration file mag update sa default value para d na ta mag `migrate:fresh` (na ma clear ang existing data sa laravel) we either create another migration to update its values or assign nlng ang default value sa register. (For this case pd ra register.) pero if u wanna practice mu create ug new migration. Here is an example of how we do it
1. ) create new migration via `php artisan make:migration update_thumbnail_column_default_on_students_table --table=students`
2. Then once mu create na na siya ug new migration add `table->string('thumbnail')->default('/storage/app/thumbnails/defaultProfile.jpg')->change();`
3. then run `php artisan migrate` to apply newly created migrations to DB.

Reply:
Sir, for the thumbnail I've made it optional, because once the user will be able to access the profile page, he/she can update the thumbnail. My plan is to have a default image in frontend. If thumbnail is null then it will display the default image, if not, then the stored thumbnail will be used.

For the empty files, If that is within the redux directory, I need them Sir, because I will implement redux approach in fetching data in integrating dashboard page and api..